### PR TITLE
Added `callable_annotation` to the base flow

### PR DESCRIPTION
**Pull request by PerisicJere**
```
"""fy
@callable
base flow HelloWorld -> None:
"""
```

### CODE REVIEW

File: library/src/fy_library/domain/fy_py_template_models.py [hunk 1]
```diff
@@ -60,6 +60,7 @@ class BaseFlowTemplateModel(BaseTemplateModel):
     abstract_property_mixins: List[AbstractPropertyModel]
     abstract_method_mixins: List[AbstractMethodModel]
     property_setters: List[ParsedAbstractPropertyFyPyFile]
+    callable_annotation: str
```
Comment (vinkobuble) [inline library/src/fy_library/domain/fy_py_template_models.py:63]: bool

File: library/src/fy_library/domain/parsed_fy_py_file.py [hunk 2]
```diff
@@ -68,6 +69,7 @@ def entity_key(self) -> str:
 class ParsedBaseFlowFyPyFile(ParsedFyPyFile):
     file_type: Literal[ParsedFyPyFileKind.BASE_FLOW] = ParsedFyPyFileKind.BASE_FLOW
     base_flow_name: PythonEntityName
+    annotations: List[Annotation] | None
```
Comment (vinkobuble) [inline library/src/fy_library/domain/parsed_fy_py_file.py:72]: List can be empty.

File: library/src/fy_library/mixins/property/filtered_mixin_imports/using_remove_existing_imports_fy.py [hunk 3]
```diff
@@ -59,7 +59,7 @@ def _filtered_mixin_imports(self) -> List[str]:
 
         user_imports_results = []
         for user_import in self._parsed_fy_py_file.user_imports.split("\n"):
-            if user_import == "":
+            if user_import == "" or user_import == "@callable":
```
Comment (vinkobuble) [inline library/src/fy_library/mixins/property/filtered_mixin_imports/using_remove_existing_imports_fy.py:62]: revert

**Review by vinkobuble — APPROVED** [review]

File: library/src/fy_library/domain/fy_py_template_models.py (hunk 1 above)
Comment (PerisicJere) [reply library/src/fy_library/domain/fy_py_template_models.py:63]: Changed from `str` to `bool`

File: library/src/fy_library/domain/parsed_fy_py_file.py (hunk 2 above)
Comment (PerisicJere) [reply library/src/fy_library/domain/parsed_fy_py_file.py:72]: `None` removed

**Pull request merged**